frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/cloud-init-tab/cloud-init-tab.tsx

```diff
@@ -210,6 +211,13 @@ const CloudInitTabComponent: React.FC<ResultTabComponentProps> = ({
     }
     cloudInitData.set(key, value);
 
+    if (key === CloudInitDataFormKeys.HOSTNAME) {
```

Comment:
I don't think we should sync these two (the user doesn't know it is changing values in the first step). It is not even properly two-way synced (hostname doesn't change cloudinit).

Reply:
we are keeping this after a discussion to keep the behaviour of the old wizard